.github/workflows/spack_cpu_build.yaml

```diff
@@ -118,7 +118,7 @@ jobs:
             config:
               install_tree:
                 root: /opt/spack
```

Comment:
@Jayapreethi @paulrigor we can also change the installation location if that's worth trying